### ISSUE TITLE
GBFR doesnt open with command

### ISSUE THREAD

**Issue by PrinzVelvet** [dvorak321/gbfr-parser#9]
when I try to use the linked opening through steam the cmd prompt opens closes and then gbf doesnt finish opening.

"C:\Users\jmaly\AppData\Local\Programs\Python\Python311\steam_win_start.cmd" %command%

the Prompt goes here bc my program has to be in the python folder otherwise I get this 
![image](https://github.com/dvorak321/gbfr-parser/assets/47064825/0aaa1a9c-24e5-42ed-9c25-ced8a7bb0819)
when I try to open it from the your program being in the documents folder. I am not really sure what is happening.

**Comment by T-kON99** [comment]
Is Python installed properly (including PATH env)? Can you try with `uac_start.cmd` instead?

**Comment by PrinzVelvet** [comment]
> Is Python installed properly (including PATH env)? Can you try with `uac_start.cmd` instead?

I managed to fix the path env after some research. So now I can open the CMD from the following path
C:\Users\jmaly\Documents\gbfr-parser.2024.2.24a
with 0 issue manually. However when I try to put in the command into steam to open them together it still wont open granblue.

**Comment by T-kON99** [comment]
I realized you put the wrong steam launch option, it should be `C:\Users\jmaly\Documents\gbfr-parser.2024.2.24a` instead of `C:\Users\jmaly\AppData\Local\Programs\Python\Python311`

**Comment by PrinzVelvet** [comment]
I tried both actually and neither worked i've tried the commands

"C:\Users\jmaly\Documents\gbfr-parser.2024.2.24a\steam_win_start.cmd" %command%
and
"C:\Users\jmaly\AppData\Local\Programs\Python\Python311\steam_win_start.cmd" %command% 
(when I had the program sitting in the python folder)

and neither of the two launch commands work they just open the cmd, it closes instantly and then then Granblue Play button goes from launching back to Play and nothing opens.

**Comment by PrinzVelvet** [comment]
it currently looks like so
C:\Users\jmaly\Documents\gbfr-parser.2024.2.24a
![image](https://github.com/dvorak321/gbfr-parser/assets/47064825/9189101e-ca12-42dc-be98-14ca6bc38ab8)
steam launch options
![image](https://github.com/dvorak321/gbfr-parser/assets/47064825/582828db-80d7-4cab-a726-0b7fcbe30abc)

**Comment by T-kON99** [comment]
If you were to click `uac_start.cmd` directly and or `steam_win_start` directly, without putting it in steam launch option, does it work?

**Comment by PrinzVelvet** [comment]
Yeah both work independently as long as they aren't in the launch settings
